plugins-extra/query_enhancements/server/routes/query_assist/agents.ts

```diff
@@ -0,0 +1,62 @@
+import { ApiResponse } from '@opensearch-project/opensearch';
+import { RequestBody, TransportRequestPromise } from '@opensearch-project/opensearch/lib/Transport';
+import { RequestHandlerContext } from 'src/core/server';
+
+const ML_COMMONS_API_PREFIX = '/_plugins/_ml';
```

Comment:
should we place this here: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/feature/discover-next/plugins-extra/query_enhancements/common/index.ts